guides/get-started-microservices-on-kubernetes/simple-microservice-database-part1.adoc

```diff
@@ -0,0 +1,413 @@
+                </feature-pack>
+                <feature-pack>
+                    <location>org.wildfly:wildfly-datasources-galleon-pack:{version-wildfly-datasources-galleon-pack}</location>
+                </feature-pack>
+            </feature-packs>
+            <layers>
+                <layer>cloud-server</layer>
+                <layer>postgresql-datasource</layer>
+            </layers>
+        </configuration>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>package</goal>
+                </goals>
+            </execution>
+        </executions>
+    </plugin>
+----
+
+=== Java Classes
+
+Add the following classes to the project:
+
+.org.wildfly.examples.model.TestTable   :
+[source,java]
+----
+package org.wildfly.examples.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+
+import java.util.Objects;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@SuppressWarnings("serial")
+@Entity
+@XmlRootElement(name = "test-table")
+public class TestTable {
+
+	@Id
+	@GeneratedValue(strategy = IDENTITY)
+	private Long id;
+
+	private String field1;
+
+	private String field2;
+
+	public Long getId() {
+		return id;
+	}
+
+	public void setId(Long id) {
+		this.id = id;
+	}
+
+	@XmlElement
+	public String getField1() {
+		return field1;
+	}
+
+	public void setField1(String field1) {
+		this.field1 = field1;
+	}
+
+	@XmlElement
+	public String getField2() {
+		return field2;
+	}
+
+	public void setField2(String field2) {
+		this.field2 = field2;
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o) return true;
+		if (o == null || getClass() != o.getClass()) return false;
+		TestTable testTable = (TestTable) o;
+		return Objects.equals(id, testTable.id) && Objects.equals(field1, testTable.field1) && Objects.equals(field2, testTable.field2);
+	}
+
+	@Override
+	public int hashCode() {
+		return Objects.hash(id, field1, field2);
+	}
+}
+----
+
+.org.wildfly.examples.model.TestTableResource :
+[source,java]
+----
+package org.wildfly.examples.service;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.persistence.PersistenceContextType;
+import jakarta.transaction.UserTransaction;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.Response;
+import org.wildfly.examples.model.TestTable;
+
+import java.util.List;
+
+@RequestScoped
+@Path("/test-table")
+public class TestTableResource {
+
+	@PersistenceContext(type = PersistenceContextType.EXTENDED,unitName = "primary")
```

Comment:
There is no need to use an extended persistent context since you are using a request scope bean. This can be simplified and use the default one.

Reply:
Thank you @yersan : fixed!